doc/basic_configuration.rst

```diff
@@ -55,17 +61,32 @@ In *all* build files::
     emails:
     - your_email@example.com
 
+Note that you need to have a local smtp service configured for email notifications.
```

Comment:
This is part of the buildfarm deployment instructions. Maybe reference them here?

Reply:
Thanks! Will add the reference.